crates/ruff/src/importer/mod.rs

```diff
@@ -73,6 +75,52 @@ impl<'a> Importer<'a> {
         }
     }
 
+    /// Move an existing import into a `TYPE_CHECKING` block.
+    ///
+    /// If there are no existing `TYPE_CHECKING` blocks, a new one will be added at the top
+    /// of the file. Otherwise, it will be added after the most recent top-level
+    /// `TYPE_CHECKING` block.
+    pub(crate) fn to_typing_import(
```

Comment:
Nit: I would expect `to_typing_import` to return a `TypingImport`. Maybe `to_typing_import_edits`? 

Can we document the return type? Why does it return a tuple? What's the first/second edit? Or maybe return a struct here with named fields (and call it `TypingImport` and you're good with `to_typing_import` ;))